Cargo.toml

```diff
@@ -10,7 +10,8 @@ aurora-engine = { git = "https://github.com/aurora-is-near/aurora-engine.git", t
 aurora-engine-precompiles = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0", features = ["std"] }
 aurora-engine-sdk = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0", features = ["std"] }
 aurora-engine-transactions = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0", features = ["std"] }
-aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0", features = ["std"] }
+aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0", features = ["std", "serde"] }
+rjson = { git = "https://github.com/aurora-is-near/rjson", rev = "cc3da949", default-features = false, features = ["integer"] }
```

Comment:
I suggest not using this `serde_json` is much better.

Reply:
Sorry for late response, I was making docs and I check now. I see, I think I see `aurora-engine` also uses `rjson` somewhere as well. Can I update dependancies?

Reply:
The engine uses `rjson` because it is `nostd`. Once the engine starts using the standard library again we can remove the `rjson` dependency there. This CLI has used the standard library from the beginning and so does not need `rjson`.